--- a/mpv.conf
+++ b/mpv.conf
@@ -89,8 +89,8 @@
  # 音频 #
  ########
 
- ao = wasapi
-#audio-device = "wasapi/{xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx}"
+ ao = pipewire
+#audio-device = "pipewire/{xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx}"
 #audio-exclusive = yes
 
  volume = 100   # 注意：此参数当前在 script-opts.conf 文件中存在关联处理（参见 “保存全局属性并恢复” 部分）
@@ -160,23 +160,23 @@
 #blend-subtitles = yes
  sub-hdr-peak = 100
 
- sub-font = "LXGW WenKai Mono Lite"
+ sub-font = "思源黑体 Bold"
  sub-font-size = 44
 #sub-scale-with-window = no
  sub-scale-signs = yes
  sub-outline-size = 4
  sub-blur = 1
- sub-color = "#FFD766"
+ sub-color = "#FFFFFF"
 #sub-back-color = "#641C1C1C"
 #sub-shadow-offset = 2
- sub-use-margins = yes
+ sub-use-margins = no
 
 #sub-ass-use-video-data = aspect-ratio
 #sub-ass-video-aspect-override  = 16:9
 
 #sub-ass-override = force
 #sub-ass-scale-with-window = yes
- sub-ass-force-margins = yes
+ sub-ass-force-margins = no
 #sub-ass-style-overrides = Fontname=LXGW WenKai Mono Lite
 
  image-subs-hdr-peak = 10000
@@ -200,5 +200,5 @@
 
  screenshot-tag-colorspace = no
  screenshot-high-bit-depth = no
- screenshot-template = "~~desktop/MPV-%P-N%n"
+ screenshot-template = "~/图片/MPV/%P-N%n"
 
--- a/input_uosc.conf
+++ b/input_uosc.conf
@@ -3,7 +3,7 @@
 
 
 #                  script-binding uosc/open-file                         #! 加载 > ※ 文件浏览器
- Ctrl+Shift+v      update-clipboard text; loadfile ${clipboard/text}     #! 加载 > 剪贴板URL
+ Ctrl+Shift+v      loadfile ${clipboard/text}     #! 加载 > 剪贴板URL
 #                  script-binding uosc/load-subtitles                    #! 加载 > ※ 导入 字幕轨
 
 #                  script-binding uosc/playlist                          #! 导航 > ※ 播放列表
--- a/script-opts.conf
+++ b/script-opts.conf
@@ -85,7 +85,7 @@
  script-opts-append = uosc-audio_types=aac,ac3,ape,dsf,dts,flac,m4a,mka,mp3,ogg,opus,wav,wma,wv
  script-opts-append = uosc-image_types=apng,avif,bmp,gif,jfif,jpeg,jpg,jxl,png,svg,tif,tiff,webp
  script-opts-append = uosc-subtitle_types=ass,idx,lrc,mks,pgs,sup,srt,ssa,txt,vtt
- script-opts-append = uosc-default_directory={drives}
+ script-opts-append = uosc-default_directory=~~
  script-opts-append = uosc-languages=slang
  script-opts-append = uosc-idlemsg=mpv-lazy
 #script-opts-append = uosc-custom_font=LXGW WenKai Mono
